pkgs/buildCatppuccinPort/package.nix

```
@@ -17,20 +17,21 @@ stdenvNoCC.mkDerivation (
 
   args'
   // {
-    version = args'.version or (builtins.substring 0 7 finalAttrs.src.rev);
+    pname = args'.pname or "ctp-${args'.port}";
+    version = args'.version or "0+${builtins.substring 0 7 finalAttrs.src.rev}";
```

I do believe `nvd` is intended for semver, so it would not be better to provide this structure.
```suggestion
    version = args'.version or "0.0.0+${builtins.substring 0 7 finalAttrs.src.rev}";
```

---

`nvd` has very lax version parsing rules, and the `0+` here merely serves to indicate the start of the version number for the benefit of tools such as `nvd` (since [versions must start with digits](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)). If the last modified dates were written into `sources.json`,  however, the more informative `"0-unstable-1970-01-01"` format could be used; would this usage justify the additional data?

---

Userstyles moved to [calver](https://calver.org/) which may actually be a good idea here too. And nvd would be able to detect that too! Though it would make the versions look funny being at `2025.02.31` for example.